src/types/label.js

```diff
@@ -131,21 +131,23 @@ function drawCallout(ctx, element) {
     return;
   }
   const callout = options.callout;
-  const {separatorStart, separatorEnd} = getCalloutSeparatorCoord(element, calloutPosition);
-  const {sideStart, sideEnd} = getCalloutSideCoord(element, calloutPosition, separatorStart);
+
   ctx.save();
   ctx.beginPath();
   const stroke = setBorderStyle(ctx, callout);
+  if (!stroke) {
+    return ctx.restore();
+  }
+  const {separatorStart, separatorEnd} = getCalloutSeparatorCoord(element, calloutPosition);
+  const {sideStart, sideEnd} = getCalloutSideCoord(element, calloutPosition, separatorStart);
```

Comment:
Are these changes somehow related to the bug, or just an optimization?

One less line and maybe one point less complex in CC without the return (`if (stroke) { ...draw... }`). Though its going to nest one level more, so the complexity might be the same.